.github/workflows/go.yml

```diff
@@ -18,8 +18,7 @@ jobs:
     - name: Format
       run: make fmt
 
-    - name: Lint
-      run: make lint
-
     - name: Test
+      env:
+        GOPROXY: https://proxy.golang.org
```

Comment:
cool!

Reply:
Supposedly people say it really speeds up the go module downloads, and since we have a limited amount of time per month, i figured every little bit would help

Reply:
:+1: